radio/src/Makefile

```diff
@@ -850,6 +856,8 @@ ifeq ($(PCB), $(filter $(PCB), SKY9X 9XRPRO AR9X))
   ifeq ($(RTCLOCK), YES)
     CPPDEFS += -DRTCLOCK
     CPPSRC += rtc.cpp targets/sky9x/rtc_driver.cpp
+  else ifeq ($(PCB), SKY9X)
+    CPPSRC += targets/sky9x/rtc_driver.cpp
```

Comment:
Yeah, but on Sky9x RTC is always managed by/part of the coproc...
Although it might actually change if we ever implement support for optional RTC on AR9x.

Still, the currently duplicated code isn't very clean...